castai/client.go

```diff
@@ -109,6 +110,21 @@ type client struct {
 	binVersion        config.SecurityAgentVersion
 }
 
+func (c *client) UpdateImageScanStatus(ctx context.Context, req *ImagesScanRequest) error {
```

Comment:
Extend our isolated kvisor e2e and assert that you got this request. See https://github.com/castai/kvisor/blob/ba22c3740905633df9ecff5d0fbe987eb6dbd7e7/e2e/e2e.go#L197